src/utils_project.js

```diff
@@ -284,8 +291,9 @@ go.utils_project = {
                         'Connection': ['close']
                     }
                 });
+
                 return http
-                    .head(voice_url)
+                    .head(voice_url[0])
```

Comment:
@NicolaasDieBaas if the length of voice_url is 1 (e.g. it's just plain) then check 0, if not ignore the first (i.e. it's the standard error) and check 1. Sorry, not the clearest instruction!